docs/getting-started.txt

```diff
@@ -180,130 +188,178 @@ hyperparameters on a single slot.
 
-The loss curve is similar to the single-GPU experiment in the previous exercise but the time to
-complete the trial is reduced by about half.
+#. To view the WebUI dashboard, enter the cluster address in your browser address bar, accept the
+   default ``determined`` username, and click **Sign In**. A password is not required.
+
+#. Click the **Experiment** tile to view the experiment’s trial display. The loss curve is similar
+   to the single-GPU experiment in the previous exercise but the time to complete the trial is
+   reduced by about half.
 
 *********************************
  Run a Hyperparameter Tuning Job
 *********************************
 
-Hyperparameter searches involve multiple trials or model variations per experiment. This example
-demonstrates hyperparameter search using the ``adaptive.yaml`` configuration file. The parameter
-settings tell the search algorithm the ranges to explore for each hyperparameter. This example model
-uses a fixed batch size and searches on dropout size, filters, and learning rate. The searcher name
-is set to ``adaptive_asha`` and ``max_trials`` is set to ``16``, indicating which search algorithm
-to use and how many model configurations to explore.
-
-#. Submit the experiment specifying the configuration file:
+This example demonstrates hyperparameter search. The example uses the ``adaptive.yaml``
+configuration file, which is similar to the ``const.yaml`` file in the first example but includes
+additional hyperparameter settings:
+
+.. code:: yaml
+
+   hyperparameters:
+     global_batch_size: 64
+     learning_rate:
+       type: double
+       minval: .0001
+       maxval: 1.0
+     n_filters1:
+       type: int
+       minval: 8
+       maxval: 64
+     n_filters2:
+       type: int
+       minval: 8
+       maxval: 72
+     dropout1:
+       type: double
+       minval: .2
+       maxval: .8
+     dropout2:
+       type: double
+       minval: .2
+       maxval: .8
+
+Hyperparameter searches involve multiple trials or model variations per experiment. The
+configuration settings tell the search algorithm the ranges to explore for each hyperparameter.
+
+The ``adaptive_asha`` search method and maximum number of trials, max_trials` are also specified:
+
+.. code:: yaml
+
+   searcher:
+     name: adaptive_asha
+     metric: validation_loss
+     smaller_is_better: true
+     max_trials: 16
+     max_length:
+       batches: 937
+
+This example uses a fixed batch size and searches on dropout size, filters, and learning rate. The
+``max_trials`` setting of ``16`` indicates how many model configurations to explore.
+
+#. Create and run the experiment:
 
    .. code:: bash
 
       det experiment create adaptive.yaml .
 
-#. Enter your cluster address in the browser.
+#. To view the WebUI dashboard, enter your cluster address in the browser address bar, accept the
+   default determined username, and click **Sign In**. No password is required.
```

Comment:
didn't we already say this in this same doc?

Reply:
I don't mind repeating if it saves the user from making a temporal leap for this kind of content. "Spacial adjacency trumps temporal stacking" - Tufte. Which is also why I'm moving the env var mention to where it applies.